packages/story-editor/src/app/story/effects/useHistoryEntry.js

```diff
@@ -27,8 +27,14 @@ import { useHistory } from '../../history';
 import deleteNestedKeys from '../utils/deleteNestedKeys';
 
 // Changes to these properties of elements do not create a new history entry.
```

Comment:
Since you mentioned this only happens if _only_ this property changes and nothing else, it would be good to reflect that here in the docs

Reply:
Updated the comment.